wlog-glog/internal/marshalers/wtracing_spanmodel.go

```diff
@@ -68,6 +68,19 @@ func marshalWTracingSpanModel(key string, val interface{}) string {
 		}
 	}
 
+	if tags := span.Tags; tags != nil && len(tags) > 0 {
+		_, _ = builder.WriteString(`, tags: {`)
+		idx := 0
+		for k, v := range tags {
+			builder.WriteString(k + ": " + v)
```

Comment:
need to ignore the returns. I am actually surprised the linters don't catch this right now. Maybe because its in the loop 🤷‍♂️

Reply:
Yep that was an oversight. I usually do depend on verify to catch those issues.